GitCommands/Settings/AppSettings.cs

```diff
@@ -1910,7 +1910,7 @@ public static Color GetColor(AppColor name)
             return SettingsContainer.GetColor(name.ToString().ToLowerInvariant() + "color", AppColorDefaults.GetBy(name));
         }
 
-        public static void SetEnum<T>(string name, T value)
+        public static void SetEnum<T>(string name, T value) where T : Enum
```

Comment:
Added an `Enum` type constraint here, which tells the analysis that values of `T` can never be null.